src/NServiceBus.Core/Features/FeatureActivator.cs

```diff
@@ -160,7 +160,7 @@ static bool DirectedCycleExistsFrom(Node node, Node[] visitedNodes)
             return false;
         }
 
-        bool ActivateFeature(FeatureInfo featureInfo, List<FeatureInfo> featuresToActivate, IConfigureComponents container, PipelineSettings pipelineSettings, RoutingComponent routing)
+        bool ActivateFeature(FeatureInfo featureInfo, List<FeatureInfo> featuresToActivate, IConfigureComponents container, PipelineSettings pipelineSettings, RoutingComponent routing, ReceiveConfiguration receiveConfiguration)
```

Comment:
nitpicking: if we adopt this schema then we should refactor the `RoutingComponent` so that it also consists of `Configuration` and `Runtime` parts so that the components look and behave similar.

Reply:
I agree with @SzymonPobiega